src/config/containers-and-vms/chroot.md

```diff
@@ -0,0 +1,90 @@
+# Creating and using chroots and containers
+
+chroots and containers can be set up and used for many purposes, including:
+
+- running glibc software on musl (and vice versa)
+- running software in a more controlled or sandboxed environment
+- creating a rootfs for bootstrapping a system
+
+## Chroot Creation
+
+Create a directory that will contain the chroot, then install a base system in
+it via the `base-voidstrap` package:
+
+```
+$ XBPS_TARGET_ARCH=<chroot_arch> xbps-install -M -r <chroot_dir> -R <repository> base-voidstrap
+```
+
+The `<repository>` varies depending on architecture:
+
+- `aarch64`, `aarch64-musl`:
+   `https://repo-default.voidlinux.org/current/aarch64`
+- any other `musl` architecture:
+   `https://repo-default.voidlinux.org/current/musl`
+- any other architecture: `https://repo-default.voidlinux.org/current`
+
+Alternatively, this can be done automatically with
+[`xvoidstrap(1)`](https://man.voidlinux.org/xvoidstrap.1) (from `xtools`):
+
+```
+# XBPS_ARCH=<chroot_arch> xvoidstrap <chroot_dir> base-voidstrap
+```
+
+## Chroot Usage
+
+### xchroot
```

Comment:
here, the xtools script has its separate section and is mentioned before the manual method, above  in Chroot Creation, it is mentioned at the end and doesn't have a separate section. Do you think it would be a good idea to unify this? I would propose adding a section called xvoidstrap to Chroot Creation before the manual instructions and put the manual instructions to a section called Manual Method similar to how it's done in Chroot Usage.